java/src/jmri/jmrit/beantable/PropertyAddEditAction.java

```diff
@@ -0,0 +1,94 @@
+package jmri.jmrit.beantable;
+
+import java.awt.event.ActionEvent;
+import java.util.Vector;
+
+import javax.swing.*;
+import javax.swing.event.*;
```

Comment:
Static analysis fails on this line since `javax.swing.event.*` is never used.

Reply:
Thanks!  Both fixed.